README.md

```diff
@@ -25,7 +25,7 @@ Music-based data streams often have repeated elements that build on each other,
 
 ### Audience
 
-Who is our audience? Check with KK.
+The target audience is people who are working with sequential data where repetitions have meaning: computational researchers, advanced undergraduate students who can be seen as younger industry esperts, etc. For example, people who are both music lovers and know computer science are our audiences since they will be interested in knowing the implication and application of Python in music.
```

Comment:
I think you misspelled "experts"

Reply:
Also I think the phrase "people who are both music lovers and know computer science" can be rephrased as either "people who are both music lovers and computer science enthusiasts" or "people who both love music and know computer science" or something like that for grammatical consistency, if that makes sense.

Reply:
Also I think "implication" should go after "application" if that makes any sense too, because "implication" is like something that goes after/the effect?

Reply:
But generally the idea looks good!

Reply:
> Also I think "implication" should go after "application" if that makes any sense too, because "implication" is like something that goes after/the effect?

I agree. And I am not sure if we can add a word implementation here. So the sentence would be implementation, application, and implication of Python in music. (I am trying to add words to describe the whole process)